### DISCUSSION TITLE
Tasks fails at random - no logs

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#29634] [Q&A] [answered]
Hello,

I  have from time to time a random task failing without a log file. The previous task is always successful.

```
*** Log file does not exist: /opt/airflow/logs/dag_id=dag_file/run_id=scheduled__2023-02-18T05:00:00+00:00/task_id=standard_live/attempt=1.log
*** Fetching from: http://:8793/log/dag_id=dag_file/run_id=scheduled__2023-02-18T05:00:00+00:00/task_id=standard_live/attempt=1.log
*** Failed to fetch log file from worker. Request URL is missing an 'http://' or 'https://' protocol.
```

I am using airflow 2.5.1 with docker-compose and the tasks are docker operators.
There are no errors in the log files for the dag,scheduler,webserver etc.

How can I  find what's going on?

**Comment by ghost** [comment]
Every 120sec  [objsync](https://github.com/scribd/objinsync) is syncing the dags.
Could this causing the problem?

**Reply by potiuk** [reply]
If it deletes files when they got updated and scheduler scans them at the same time, yes it can cause problems. If you are unsure how your syncing works - use one of the recommented solutins (see our Helm chart for good description on how to sync DAGs.  I always recommend GitSync because it performs atomic updates

**Reply by potiuk** [reply]
See https://medium.com/apache-airflow/shared-volumes-in-airflow-the-good-the-bad-and-the-ugly-22e9f681afca

**Comment by ghost** [comment, marked as answer]
The problem was that  I was running a lot of s3 requests for a long period of time.
This might have caused objsync to not be able to sync some dag files in a consistent way.
When I have stopped the s3 requests everything goes back to normal .
